### PR TITLE
PR 1 - Update Readme and Images - Classic to T2.1

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**### Sample List - 
bot-sso-adaptivecard - csharp
bot-sso-adaptivecard - nodejs
graph-change-notification - csharp
graph-change-notification - nodejs
graph-channel-lifecycle - csharp
graph-channel-lifecycle - nodejs
meetings-details-tab - nodejs
meetings-transcription - csharp
meetings-transcription - nodejs
tab-channel-context**

### CODE REVIEW

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/6.MeetingTranscriptCard.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/csharp/MeetingTranscription/Images/6.MeetingTranscriptCard.png:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/7.TranscriptTaskModule.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/csharp/MeetingTranscription/Images/7.TranscriptTaskModule.png:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/MeetingsTranscriptionGif.gif
Comment (Pawank-MSFT) [inline samples/meetings-transcription/csharp/MeetingTranscription/Images/MeetingsTranscriptionGif.gif:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/Policy.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/csharp/MeetingTranscription/Images/Policy.png:1]: Hide account, AppIds and other information

File: samples/meetings-transcription/nodejs/Images/3.JoinMeeting.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/nodejs/Images/3.JoinMeeting.png:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/nodejs/Images/6.MeetingTranscriptCard.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/nodejs/Images/6.MeetingTranscriptCard.png:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/nodejs/Images/7.TranscriptTaskModule.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/nodejs/Images/7.TranscriptTaskModule.png:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/nodejs/Images/MeetingTranscript.gif
Comment (Pawank-MSFT) [inline samples/meetings-transcription/nodejs/Images/MeetingTranscript.gif:1]: Please hide your and Jegadeesh name.

File: samples/meetings-transcription/nodejs/Images/Policy.png
Comment (Pawank-MSFT) [inline samples/meetings-transcription/nodejs/Images/Policy.png:1]: hide account and app related information

**Review by Pawank-MSFT — COMMENTED** [review]
some small comments please fix

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/6.MeetingTranscriptCard.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/csharp/MeetingTranscription/Images/6.MeetingTranscriptCard.png:1]: Fixed

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/7.TranscriptTaskModule.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/csharp/MeetingTranscription/Images/7.TranscriptTaskModule.png:1]: Fixed

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/MeetingsTranscriptionGif.gif
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/csharp/MeetingTranscription/Images/MeetingsTranscriptionGif.gif:1]: Fixed

File: samples/meetings-transcription/nodejs/Images/6.MeetingTranscriptCard.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/nodejs/Images/6.MeetingTranscriptCard.png:1]: Fixed

File: samples/meetings-transcription/nodejs/Images/7.TranscriptTaskModule.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/nodejs/Images/7.TranscriptTaskModule.png:1]: Fixed

File: samples/meetings-transcription/nodejs/Images/Policy.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/nodejs/Images/Policy.png:1]: Fixed

File: samples/meetings-transcription/nodejs/Images/MeetingTranscript.gif
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/nodejs/Images/MeetingTranscript.gif:1]: Fixed

File: samples/meetings-transcription/csharp/MeetingTranscription/Images/Policy.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/csharp/MeetingTranscription/Images/Policy.png:1]: Fixed

File: samples/meetings-transcription/nodejs/Images/3.JoinMeeting.png
Comment (Harikrishnan-MSFT) [reply samples/meetings-transcription/nodejs/Images/3.JoinMeeting.png:1]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
Build - Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8845093424

image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**